recipe/meta.yaml

```diff
@@ -6,22 +6,23 @@ package:
 
 source:
   url: https://github.com/ibis-project/ibis/archive/{{ version }}.tar.gz
-  sha256: 33ba671fa85accebbe7ab91a2f4eb575b485ae39396e3273dfbeaef35df948dc
+  sha256: 36e0b2694a4e88a1406756eb32b21df7a625009f83bdfed012ab0159c7b49117
 
 build:
-  number: 1
+  number: 2
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
-  noarch: python
+  # uncomment noarch when pymapd and pyspark issues are fixed for py38
+  # noarch: python
 
 requirements:
   host:
     - pip
-    - python >=3
+    - python
     - setuptools
 
   run:
     - clickhouse-driver >=0.1.3
-    - clickhouse-cityhash
+    - clickhouse-cityhash  # [not win]
```

Comment:
the feedstock windows tests worked. but not sure how is the impact for the entire backend without this package.
ibis ci doesn't test clickhouse backend